### DISCUSSION TITLE
Installing statamic in an existing laravel projects all pages 404ing

### DISCUSSION THREAD

**Discussion by joshuapphillips** [statamic/ssg#125] [Q&A]
I have installed statamic in an existing Laravel projects. I have created a pages collection but all the pages are going to 404s.

I have templates and a layout set up.

What could be causing this?

**Comment by duncanmcclean** [comment]
This isn't a question for the SSG repository (unless it's only happening after you run the SSG?)

Have you setup a route for your pages collection?
